.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-NODE_ENV=dev && node_modules/.bin/lint-staged && npm run typecheck && npm test
```

Comment:
We definitely don't want any of these, e.g. linting and typechecking (but not the test suite)?

Reply:
I find it slows things down (I know you can flag `--no-verify` but it's a bit annoying) and is run on CI anyway, so I think it's ok to get rid of. We can run them manually as part of `script/test` too. Any objections to removing it?

Reply:
Nah go for it. I trust your judgment on how much time it adds!